common/locales/en/defaultTasks.json

```diff
@@ -1,21 +1,19 @@
 {
-  "defaultHabit1Text": "Productive Work (Click the pencil to edit)",
```

Comment:
@DrStrangepork Two spaces is our convention for our code files. While json isn't exactly code and some json files do have four spaces, for consistency two spaces should be used when that'rs already done in a json file.

The en locales files are different to the other locales. Please see the _README_FIRST.md file in locales/en_gb for details.